### DISCUSSION TITLE
非K8S集群下的DeepFlow-Agent是否支持集成 OpenTelemetry 数据？

### DISCUSSION THREAD

**Discussion by mcy567** [deepflowio/deepflow#4308] [Q&A]
看官网里面 https://deepflow.io/docs/zh/agent-integration/tracing/opentelemetry/ 是通过K8S操作开启 deepflow-agent 的数据接收服务，请问针对 https://deepflow.io/docs/zh/install/legacy-host/ 这种情况部署的Agent是否支持接收OpenTelemetry 数据？ @dundun9

**Comment by Nick-0314** [comment]
可以的，应该是差不多的配置
